.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->in([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+    ])
+    ->exclude([
+        'data'
+    ])
+;
+
+return (new PhpCsFixer\Config())
+    ->setRules([
+        '@PSR12' => true,
```

Comment:
This should probably be `@PER` instead.

https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6545

Reply:
:+1: agreed!  That sound's like a better and more future-proof plan.

I forgot about the existence of PER, which has replaced PSR-12.

Nice catch, @X-Coder264!